src/app/components/feed/FeedImportDialog.js

```diff
@@ -0,0 +1,367 @@
+                />
+              </div>
+              <div className="typography-subtitle2">
+                <FormattedMessage
+                  id="feedImportDialog.addNotPart2"
+                  defaultMessage="You can import this media in a {newItemLink}, or search for an {existingItemLink}."
+                  description="Explanation on import dialog on feed item page... the default in English for 'newItemLink' is 'new item in your workspace', while for 'existingItemLink' it's 'existing item to import'."
+                  values={{
+                    newItemLink: (
+                      <span onClick={() => { setImportType('create'); }} onKeyDown={() => { setImportType('create'); }} className={styles.feedImportLink}>
+                        <FormattedMessage
+                          id="feedImportDialog.addNotPartLinkNew"
+                          defaultMessage="new item in your workspace"
+                          description="This is the text of the link to create a new item in the feed import dialog. It's used in the middle of a sentence like: You can import this media in a 'new item in your workspace'."
+                        />
+                      </span>
+                    ),
+                    existingItemLink: (
+                      <span onClick={() => { setImportType('search'); }} onKeyDown={() => { setImportType('search'); }} className={styles.feedImportLink}>
+                        <FormattedMessage
+                          id="feedImportDialog.addNotPartLinkSearch"
+                          defaultMessage="existing item to import"
+                          description="This is the text of the link to create a new item in the feed import dialog. It's used in the middle of a sentence like: You can import this media in a new item or search for an 'existing item to import'."
+                        />
+                      </span>
+                    ),
+                  }}
+                />
+              </div>
+            </div>
+          )}
+
+          {/* Third case: Search for media */}
+          { importType === 'search' && (
+            <div id="feed-import-dialog__search" className={cx(styles.feedImportSearch, mediaStyles['media-item-autocomplete-wrapper'])}>
+              <AutoCompleteMediaItem onSelect={setSelectedItem} />
+            </div>
+          )}
+        </div>
+
+        {/* Footer */}
+        <div className={dialogStyles['dialog-actions']}>
+          <ButtonMain
+            className="feed-import-dialog__close-button"
+            size="default"
+            theme="text"
+            onClick={onClose}
+            disabled={saving}
+            label={
+              <FormattedMessage
+                id="feedImportDialog.cancel"
+                defaultMessage="Cancel"
+                description="Label of a button to close the import dialog window on the feed item page."
+              />
+            }
+          />
```

Comment:
Please reuse same props as global.edit in globalStrings.js

Reply:
Done!